src/bootstrap/dist.rs

```diff
@@ -2023,17 +2023,14 @@ impl Step for RustDev {
     fn run(self, builder: &Builder<'_>) -> Option<GeneratedTarball> {
         let target = self.target;
 
-        /* run only if llvm-config isn't used */
-        if let Some(config) = builder.config.target_config.get(&target) {
-            if let Some(ref _s) = config.llvm_config {
-                builder.info(&format!("Skipping RustDev ({}): external LLVM", target));
-                return None;
-            }
+        let mut tarball = Tarball::new(builder, "rust-dev", &target.triple);
+
+        let bootstrap_outdir = &builder.bootstrap_out;
+        for file in &["bootstrap", "llvm-config-wrapper", "rustc", "rustdoc", "sccache-plus-cl"] {
+            tarball.add_file(bootstrap_outdir.join(file), "bootstrap/bin", 0o755);
         }
 
-        let mut tarball = Tarball::new(builder, "rust-dev", &target.triple);
         tarball.set_overlay(OverlayKind::LLVM);
-
         builder.ensure(crate::native::Llvm { target });
```

Comment:
now that you've added the new `Bootstrap` step, you can undo all changes to `rust-dev` and have it stay as LLVM only. You'll also need to update all the CI changes to use `bootstrap` instead of `rust-dev`. Sorry this keeps changing :( I think this should hopefully be the last time.

Reply:
I still see changes here :(